modules/pel/peltool/peltool.py

```diff
@@ -472,6 +495,8 @@ def main():
                         action='store_true', help='Show number of PELs')
     parser.add_argument('-r', '--reverse',
                         action='store_true', help='Reverse order of output')
+    parser.add_argument('-c', '--clear',
+                        dest='IDToDelete', help='Delete a PEL based on its ID')
```

Comment:
Used "pelID" for -i option.

Reply:
That doesn't matter. We can use the same variable since we're trying to get the PEL ID in both cases, right?

In other words, we'll be able to use both options -i and -c,??

Reply:
We are using this variable to know if this option was selected or not. If selected then the value passed is store in pelID variable.